compose/integrations/composable-test-cases/build.gradle.kts

```diff
@@ -22,7 +20,9 @@ allprojects {
 
         maven("https://packages.jetbrains.team/maven/p/kt/dev")
         maven("https://redirector.kotlinlang.org/maven/dev")
-//        mavenLocal()
+        if (System.getenv("TEAMCITY_VERSION") != null) {
```

Comment:
Instead of `TEAMCITY_VERSION` it is better to use a custom Gradle property `project.property("use.maven.local") == "true"` and pass it from CI.

This way we don't bound the code to TeamCity (and can run in GitHub actions if needed)

Reply:
It is still better to not the use existing parameters which purpose is different. `kotlin.version` is currently only for changing the Kotlin version. Adding a new repo is a more broad responsibility.

But if we add `dirs("build/maven-project")`, we can add it just without condition.